src/Umbraco.Forms.Integrations.Commerce.EMerchantPay/PaymentProviderWorkflow.cs

```diff
@@ -194,11 +199,9 @@ public override WorkflowExecutionStatus Execute(Record record, RecordEventArgs e
                 BusinessAttribute = new BusinessAttribute { NameOfTheSupplier = _paymentProviderSettings.Supplier },
                 TransactionTypes = new TransactionTypeDto
                 {
-                    TransactionTypes = new List<TransactionTypeRecordDto>
-                    {
-                        new TransactionTypeRecordDto { TransactionType = "authorize" },
-                        new TransactionTypeRecordDto { TransactionType = "sale" }
-                    }
+                    TransactionTypes = _paymentProviderSettings.TransactionTypes.Split(';')
```

Comment:
Am thinking here we could make at least the Umbraco 9/10 version a bit friendlier, as with `appSettings.json` we can store an array of values, rather than a `;` delimited string.  I appreciate that makes the code a little more complicated but if there's a way to manage that it would be better to make it easier to use.